Heejung/Lv2/소수 찾기.js

```diff
@@ -0,0 +1,37 @@
+// 에라토스테네스의 체
+function makePrimeNums(n) {
+  const arr = new Array(n + 1).fill(true);
+  arr[0] = false;
+  arr[1] = false;
+
+  for (let i = 2; i * i <= n; i += 1) {
+    if (arr[i]) {
+      for (let j = i * i; j <= n; j += i) {
+        arr[j] = false;
+      }
+    }
+  }
+  return arr.map((e, i) => (e ? i : false));

```

Comment:
```suggestion
  return arr.map((e, idx) => (e ? idx : false));
```
index라는 표현을 확실하게 해주기 위해 i보다는 idx나 index라고 표현해주시는게 좋을거같아요.